ambry-api/src/main/java/com.github.ambry/config/StatsManagerConfig.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.config;
+
+/**
+ * The configs for stats.
+ */
+public class StatsManagerConfig {
+
+  @Config("stats.output.file.path")
+  @Default("/tmp/stats_output.json")
+  public final String outputFilePath;
+
+  @Config("stats.publish.period.in.secs")
+  @Default("3600")
+  public final long publishPeriodInSecs;
+
```

Comment:
I assume once we deploy this code, StatsManager will collect stats once an hour and publish it to a file in /tmp. Can we add a config to enable or disable this publishing. Also keep in mind that w/o bucketing, this will bring all index files to memory every hour.